seed/data_importer/tasks.py

```diff
@@ -1138,350 +1137,6 @@ def _get_field_from_obj(field_obj, field):
     return m.hexdigest()
 
 
-def filter_duplicated_states(unmatched_states):
```

Comment:
These methods were moved to the `/data_importer/match.py` file and heavily refactored.